### ISSUE TITLE
train/test/folder image structure for fine-tuning via Inception V3 for mutation classification

### ISSUE THREAD

**Issue by monajalal** [ncoudray/DeepPATH#100]
Hi Nicolas,

Thanks a lot for your great scientific contributions.
I had a couple of questions regarding mutation classification.

I understand that you have a label file that resembles like this:
![Screenshot from 2022-03-03 17-37-21](https://user-images.githubusercontent.com/1892917/156664922-6b9f5493-e80f-417f-b602-7ed15feef035.png)

However, if I want to do transfer learning by finetuning the Inception V3, (assuming we want to classify for presence of absence of TP53), we need to have folder such as below:

-- train
----TP53
----not_TP53

-- val
----TP53
----not_TP53

-- test 
----TP53
----not_TP53

If we were doing WSI-level classification, this would have been an easy task. However, since we are doing patch-based inputs to Inception V3, I am baffled as how do you systematically keep track of these patches in a WSI level, when pouring them into these folders (I understand you do the 70/15/15 division on WSI level -- but how do you exactly pour the patch images in the folder?) as well as how do you keep track of patches decision for global average pooling or any sort of majority voting when it comes to predictions on patch-level?

Also, am I understanding correctly that you have **weak label** for the patches, meaning you assign the label of the WSI to the entire patches inside that WSI?
